### ISSUE TITLE
Request for Supplemental Data (WeekData, TerritoryCodes, TerritoryContainment)

### ISSUE THREAD

**Issue by stelgenhof** [punic/punic#46]
Currently I retrieve the following myself parsing the /common/supplemental/supplementalData.xml file, however would appreciate if you could include these into Punic:

From /supplementalData/weekData:
- weekendStart
- weekendEnd

From /supplementalData/codeMappings/territoryCodes:
- numeric
- alpha3

/supplementalData/territoryContainment/group, as I need these to identify if a country is part of the EU community.

Thanks in advance!

**Comment by mlocati** [comment]
> From /supplementalData/weekData:
> 
> - weekendStart
> - weekendEnd

That would be nice. I'll add them quite soon.

-----

> From /supplementalData/codeMappings/territoryCodes:
> 
> - numeric
> - alpha3

That would be nice. I'll add them quite soon.

-----

> I need these to identify if a country is part of the EU community.

You should already be able to do so. Since the CLDR identifier for EU is `'EU'`, you can get the list of the EU countries by calling:
```php
Punic\Territory::getChildTerritoryCodes('EU');
```

-----

PS: If you open one GitHub issue per request it would be easier to track each of them.

**Comment by stelgenhof** [comment]
Yes,  I checked and the Territory::getChildTerritoryCodes('EU') works. I overlooked that one.

**Comment by c960657** [comment]
The numeric and alpha3 territory code lookup is implemented in #138.
